StackOverflow/Controller/QuestionsTableViewController.m

```diff
@@ -0,0 +1,97 @@
+//
+//  QuestionsTableViewController.m
+//  StackOverflow
+//
+//  Created by Mali on 2017/10/04.
+//  Copyright © 2017 DVT. All rights reserved.
+//
+
+#import "QuestionsTableViewController.h"
+#import "QuestionsCell.h"
+#import "DataFetchDelegate.h"
+#import "QuestionsDataObject.h"
+#import "QuestionsData.h"
+
+
+@interface QuestionsTableViewController () <DataFetchDelegate> {
+    NSArray *dataResult;
+    QuestionsDataObject *dataObject;
+    UIActivityIndicatorView *activityIndicator;
+}
+
+@end
+
+@implementation QuestionsTableViewController
+
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    //Display a code loading indicator
+    activityIndicator = [[UIActivityIndicatorView alloc] initWithActivityIndicatorStyle:UIActivityIndicatorViewStyleGray];
+    activityIndicator.alpha = 1.0;
+    [self.view addSubview:activityIndicator];
+    activityIndicator.center = CGPointMake([[UIScreen mainScreen]bounds].size.width/2, [[UIScreen mainScreen]bounds].size.height/2);
+    [activityIndicator startAnimating];
+
+    dataObject = [[QuestionsDataObject alloc] init];
+    dataObject.questionRequest = [[QuestionRequest alloc] init];
+    dataObject.questionRequest.delegate = dataObject;
+    dataObject.delegate = self;
+    
+    [dataObject getData];
```

Comment:
Try to make your names more meaningful. Get what data from where ? Invocations on Objective C should read like sentences. This does not, do you see here how meaningless dataObject as a name is ? It tells me nothing. If the QuestionsDataObject is actually an interface to a backend question server then call it that. Then this should read something like [remoteQuestionServer fetchListOfQuestions] or something like that. It tells me more about what is going on.